spatial-tracking-explainer.md

```diff
@@ -177,6 +177,33 @@ function onSessionStarted(session) {
 }
 ```
 
+### Application-supplied transforms
+Frequently developers will want to provide an additional, artificial transform on top of the user's tracked motion to allow the user to navigate larger scenes than their tracking systems or physical space allows. An examples of this would be the common "teleportation" mechanic, where the user selects a point in the virtual scene to "jump" to, after which the selected point is treated as the new origin which all tracked motion is relative to. This effect is traditionally accomplished by mathematically combining the API-provided transform with the desired additional application transforms. It is useful to allow the API do handle it, however, because it ensures that all tracked values are transformed consistently, allowing the user's viewpoint and inputs to stay in sync.
+
+Developers can specify application-specific transforms by setting the `baseTransform` attribute of any `XRReferenceSpace`. When the `transform` is not null, any values queried using the `XRReferenceSpace` will be offset by the `translation` and `orientation` the `baseTransform` describes. The `XRReferenceSpace`'s `baseTransform` can be updated at any time and will immediately take effect, meaning that any new values that are queried with the `XRReferenceSpace` will take into account the new `baseTransform`. Previously queried values will not be altered. Changing the `baseTransform` between pose queries in a single frame is not advised, since it will cause inconsistencies in the tracking data and rendered output.
+
+When interpreting an `XRRigidTransform` the `orientation` is applied prior to the `translation`. This means that, for example, a transform that indicates a quarter rotation to the right and a 1 meter translation along -Z would place a transformed object at `[0, 0, -1]` facing to the right.
```

Comment:
This is yet another benefit to `position`/`orientation` naming - developers have a natural instinct around what it means for an entity to be positioned and oriented within a coordinate system and thus would be unlikely to get this backwards.

Reply:
Note that the explanation you provide here for how to interpret the two attributes describes an object that is placed (i.e. positioned) at (0, 0, -1) with a given orientation.  If the attributes had those two names directly, that would likely be immediately obvious to developers.

Reply:
#440 has been updated to use `position/orientation` now. Thanks again for the suggestion.